annotation-web/src/main/resources/annotation-web-context.xml

```diff
@@ -87,5 +87,10 @@
     	<beans:property name="messageSource" ref="annotation_i18n_messageSource"/>
     </beans:bean>
     
+    <beans:bean id="annotationLdSerializer" class="eu.europeana.annotation.utils.serialize.AnnotationLdSerializer">
```

Comment:
these are statefull classes they cannot be declared as beans, but instantiated in each controller method